integration-tests/tests/aws/post-questions/post-questions-happy.test.ts

```diff
@@ -0,0 +1,100 @@
+import { stackOutputs } from "../resources/cloudformation-helper";
+import { clearItems, populateTable } from "../resources/dynamodb-helper";
+import { executeStepFunction } from "../resources/stepfunction-helper";
+
+describe("post-questions-happy", () => {
+  const stateMachineInput = {
+    key: "rti-p60-employee-ni-contributions",
+    value: "100.30",
+    sessionId: "12345",
+  };
+  const testUser = {
+    sessionId: "12345",
+    nino: "AA000003D",
+  };
+
+  const clearQuestionDB = async (sessionId: string) => {
+    await clearItems(output.PersonalIdenityTable as string, {
```

Comment:
The function `clearQuestionDB` assumes that the question table is being cleared. In your case, you are also clearing the person identity table here, could you split this logic out please?

Reply:
this is what we have followed in other state machine so shall we change in all places?